tests/scripts/all.sh

```diff
@@ -1216,20 +991,6 @@ component_test_valgrind () {
 
     msg "test: main suites valgrind (Release)"
     make memcheck
-
-    # Optional part(s)
-    # Currently broken, programs don't seem to receive signals
-    # under valgrind on OS X
-
-    if [ "$MEMORY" -gt 0 ]; then
```

Comment:
With this removal, the `MEMORY` variable and the `--memory` option are no longer used.

Reply:
I left options like those in, but ignored, so that our CI can still use those options and test older versions of our software.